packages/hardhat-core/test/internal/hardhat-network/helpers/providers.ts

```diff
@@ -194,3 +197,22 @@ if (INFURA_URL !== undefined) {
     },
   });
 }
+
+if (RETHNET_BINARY !== undefined) {
+  PROVIDERS.push({
+    name: "Rethnet",
+    isFork: false,
+    isJsonRpc: false,
```

Comment:
I think this should be true?

Reply:
Yes, Hardhat includes the hash of the transaction in the stack trace, but that only works for the in-process hardhat network IIRC. That's why those particular tests are skipped.

Reply:
addressed in https://github.com/NomicFoundation/hardhat/pull/4095/commits/dfb80375229c0901963db000953f65fd39c3ce5e